std/range/primitives.d

```diff
@@ -1987,6 +1987,81 @@ ElementType!R moveAt(R, I)(R r, I i) if (isIntegral!I)
 }
 
 /**
+Pops the first element from $(D r) and returns it.
+It combines $(D front) and $(D popFront) in one method.
+
```

Comment:
`Params` and `Returns` sections needed here. See other functions in this file for examples